source/_posts/2015-06-29-how-to-build-rails-5-api-only-and-ember-application.markdown

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title: How to build Rails 5 API only and Ember application
+hero_image: /blog/images/heros/post-high.jpg
+comments: true
+author:
+  name: Santiago Pastorino
+  email: santiago@wyeworks.com
+  twitter_handle: spastorino
+  github_handle:  spastorino
+  image:  /images/team/santiago-pastorino.jpg
+  description: WyeWorks Co-Founder & CTO, Ruby on Rails Core Team Member
+published: true
+---
+
+A few days ago, following long discussions, [Rails API was merged into Rails master branch](https://github.com/rails/rails/pull/19832).
+
+Because of that, in our latest post we discussed how to build an API only application. We've also shown how to integrate it with a client-side application implemented using Backbone. If you've missed this post, check it out and learn more about [how to integrate Rails API with an Backbone application](http://wyeworks.com/blog/2015/6/11/how-to-build-a-rails-5-api-only-and-backbone-application).
+
+In this post, we are going to show how a very similar client-side application now built with Ember can be integrated with the same backend application implemented using Rails API. Like in our previous post, we are going to use the [TodoMVC application](http://todomvc.com).
```

Comment:
I would say: "client-side Ember application" instead of "client-side application now built with Ember"